content/working_with/console/widgets/_index.md

```diff
@@ -29,4 +29,4 @@ For more details about adding widgets, placing them on a page and changing confi
 
 The following widgets descriptions are listed in an alphabetical order, as they do in the widgets catalog:
 
-{{%children style="h3" description="true"%}}
+{{%children %}}
```

Comment:
To have:
![image](https://user-images.githubusercontent.com/5202105/191742656-447dd970-9b34-434f-baff-3b6f1bf269fb.png)

instead of:
![image](https://user-images.githubusercontent.com/5202105/191742701-fd51539a-3e84-4e1d-93a3-f4cda46ab7b6.png)